packages/api/src/external/carequality/command/cq-directory/create-or-update-cq-organization.ts

```diff
@@ -1,152 +0,0 @@
-import NotFoundError from "@metriport/core/util/error/not-found";
-import { CarequalityManagementAPI } from "@metriport/carequality-sdk";
-import { isOboFacility } from "../../../../domain/medical/facility";
-import { OrganizationModel } from "../../../../models/medical/organization";
-import { FacilityModel } from "../../../../models/medical/facility";
-import { out } from "@metriport/core/util/log";
-import { capture } from "@metriport/core/util/notifications";
-import { errorToString } from "@metriport/shared";
-import { makeCarequalityManagementAPI } from "../../api";
-import { CQOrganization } from "../../organization";
-import { CQOrgDetails, getCqAddress, getParsedCqOrgOrFail } from "../../shared";
-import { metriportIntermediaryOid, metriportOid } from "./create-or-update-cq-facility";
-import { metriportEmail as metriportEmailForCq } from "../../constants";
-import { metriportCompanyDetails } from "@metriport/shared";
-
-const cq = makeCarequalityManagementAPI();
-
-export async function createOrUpdateCQOrganization(
```

Comment:
Refactored this into [createOrUpdateCQOrganization](https://github.com/metriport/metriport/pull/2974/files#diff-e9818b17b0e85fbe1d8c2d0c76bef56c6d392e024b71864e81e53c7d32eedd37R25)